mathesar/tests/api/test_schema_api.py

```diff
@@ -295,14 +295,14 @@ def test_schema_delete(create_schema, client):
     }
 
 
-def test_schema_dependencies(client, create_schema):
+def test_schema_dependents(client, create_schema):
     schema_name = 'NASA Schema Dependencies'
     schema = create_schema(schema_name)
 
     response = client.get(f'/api/db/v0/schemas/{schema.id}/')
     response_schema = response.json()
     assert response.status_code == 200
-    assert response_schema['has_dependencies'] is True
+    assert response_schema['has_dependents'] is False
 
```

Comment:
We would need a test case for a schema that has dependents, in order to make sure the dependency graph is correct. Most of our current codebase treats schema as a self-contained, so for writing a test case It would be a good idea to check if any of our DDL operations has support for cross schema support and use it or create a cross schema dependency using the `alembic` operators

Reply:
As discussed during the standup, PostgreSQL doesn't support cross-schema dependents for schema objects, so adding its support will be a custom solution and a part of a separate ticket.

Added a comment about this behaviour to the schema model and the appropriate property.

Reply:
It would be good to just add a test for dependencies within the schema which depend on the schema. This assures us the API works as intended.